src/pkg/packager/creator/creator_test.go

```diff
@@ -43,17 +60,28 @@ func TestLoadPackageDefinition(t *testing.T) {
 		{
 			name:        "invalid package definition",
 			testDir:     "invalid",
-			expectedErr: "package must have at least 1 component",
+			expectedErr: "found errors in package",
 			creator:     NewSkeletonCreator(types.ZarfCreateOptions{}, types.ZarfPublishOptions{}),
 		},
 	}
+	b, err := os.ReadFile("../../../../zarf.schema.json")
+	require.NoError(t, err)
+	lint.ZarfSchema = &mockSchemaLoader{b: b}
 
 	for _, tt := range tests {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
+			cwd, err := os.Getwd()
+			require.NoError(t, err)
+			defer func() {
+				err = os.Chdir(cwd)
+				require.NoError(t, err)
+			}()
+			path := filepath.Join("testdata", tt.testDir)
+			err = os.Chdir(path)
+			require.NoError(t, err)
 
-			src := layout.New(filepath.Join("testdata", tt.testDir))
+			src := layout.New(".")
```

Comment:
Since `ValidateSchema` is doing a read on the `zarf.yaml` we need to to be in the correct directory. The Creator interface already has that assumption, otherwise compose won't work, but we aren't composing in this test. #2683 will change this hopefully, but it'll be pretty involved.